### PR TITLE
fix: alter_hooklog_payload_types

### PR DESCRIPTION
**Pull request by willnewii**
## Change Summary

nc_hook_logs_v2.payload type is boolean. but in webHookHelpers.ts ,payload type it should be text. will be can't save hook log.

https://github.com/nocodb/nocodb/blob/4d423a383077d2c25fd9903856ba857abcd50800/packages/nocodb/src/lib/noco/meta/helpers/webhookHelpers.ts#L250

https://github.com/nocodb/nocodb/blob/4d423a383077d2c25fd9903856ba857abcd50800/packages/nocodb/src/lib/noco/meta/helpers/webhookHelpers.ts#L277

## Change type

- [x] fix: (bug fix for the user, not a fix to a build script)

### CODE REVIEW

File: packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    });
+  }
+};
+
+export { up, down };
+
+/**
+ * @copyright Copyright (c) 2022, Xgene Cloud Ltd
+ *
+ * @author Wing-Kam Wong <wingkwong.code@gmail.com>
```
Comment (wingkwong) [inline packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts:25]: Don't mark my name.

**Review by wingkwong — COMMENTED** [review]
Please sign CLA.

File: packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts (hunk 1 above)
Comment (willnewii) [reply packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts]: done

File: packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import Knex from 'knex';
+import { MetaTable } from '../../utils/globals';
+
+const up = async (knex: Knex) => {
+  if (knex.client.config.client !== 'sqlite3') {
```
Comment (wingkwong) [inline packages/nocodb/src/lib/noco/migrationsv2/nc_016_alter_hooklog_payload_types.ts:5]: Can you explain why sqlite3 is excluded?

**Review by wingkwong — APPROVED** [review]

**Pull request merged**